pipeline_steps/common.groovy

```diff
@@ -21,6 +21,9 @@ def install_ansible(){
     cp -r /usr/lib64/python2.6/site-packages/selinux .venv/lib64/python2.7/site-packages/ ||:
     source .venv/bin/activate
 
+    # UG-613 change TMPDIR to directory with more space
+    export PREV_TMPDIR=\$TMPDIR
+    export TMPDIR="/var/tmp"
```

Comment:
I'd go with /var/lib/jenkins/tmp as thats the largest partition. Probably needs creating though :/